dir.props

```diff
@@ -13,8 +13,8 @@
   <PropertyGroup>
     <MajorVersion>2</MajorVersion>
     <MinorVersion>0</MinorVersion>
-    <PatchVersion>0</PatchVersion>
-    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
+    <PatchVersion>1</PatchVersion>
```

Comment:
Looks like DependencyModel & PlatformAbstractions are still 2.0.0-servicing, but everything else is 2.0.1-servicing. I'll see if I can figure out where they have their package version defined.